### PR TITLE
store/tikv: remove kv.Version from package store/tikv

### PR DESCRIPTION
**Pull request by AndreMouche**
Signed-off-by: AndreMouche

### What problem does this PR solve?
This PR removes the usage of `kv.Version` from `tikv` package
Part of https://github.com/pingcap/tidb/issues/22513

### Check List 
Tests 
- Unit test

### Release note 
- No release note

### CODE REVIEW

**Review by andylokandy — APPROVED** [review]

File: store/tikv/interface.go [hunk 1]
```diff
@@ -51,15 +50,10 @@ type Storage interface {
...
 
-	// GetSnapshot gets a snapshot that is able to read any data which data is <= ver.
-	// if ver is MaxVersion or > current max committed version, we will use current version for this snapshot.
-	GetSnapshot(ver kv.Version) kv.Snapshot
 	// Close store
 	Close() error
 	// UUID return a unique ID which represents a Storage.
 	UUID() string
-	// CurrentVersion returns current max committed version with the given txnScope (local or global).
-	CurrentVersion(txnScope string) (kv.Version, error)
```
Comment (disksing) [inline store/tikv/interface.go:62]: I think we can keep a `GetTimestamp(txnScope string) (uint64, error)` method.

File: store/tikv/txn.go [hunk 2]
```diff
@@ -80,7 +80,7 @@ func newTiKVTxn(store *KVStore, txnScope string) (*KVTxn, error) {
 
 // newTiKVTxnWithStartTS creates a txn with startTS.
 func newTiKVTxnWithStartTS(store *KVStore, txnScope string, startTS uint64, replicaReadSeed uint32) (*KVTxn, error) {
-	ver := kv.NewVersion(startTS)
+	ver := startTS
```
Comment (disksing) [inline store/tikv/txn.go:83]: you can use startTs directly

File: store/tikv/snapshot.go [hunk 3]
```diff
@@ -87,10 +88,10 @@ type tikvSnapshot struct {
 }
 
 // newTiKVSnapshot creates a snapshot of an TiKV store.
-func newTiKVSnapshot(store *KVStore, ver kv.Version, replicaReadSeed uint32) *tikvSnapshot {
+func newTiKVSnapshot(store *KVStore, ver uint64, replicaReadSeed uint32) *tikvSnapshot {
```
Comment (disksing) [inline store/tikv/snapshot.go:91]: how about rename `ver` to `ts`?

File: store/tikv/kv.go [hunk 4]
```diff
@@ -213,7 +213,7 @@ func (s *KVStore) beginWithExactStaleness(txnScope string, prevSec uint64) (*KVT
 
 // GetSnapshot gets a snapshot that is able to read any data which data is <= ver.
 // if ver is MaxVersion or > current max committed version, we will use current version for this snapshot.
-func (s *KVStore) GetSnapshot(ver kv.Version) kv.Snapshot {
+func (s *KVStore) GetSnapshot(ver uint64) kv.Snapshot {
```
Comment (disksing) [inline store/tikv/kv.go:216]: rename `ver` to `ts`?

**Review by disksing — APPROVED** [review]

**Pull request merged**